### PR TITLE
New Test for media alternatives for text

### LINKED ISSUE #382: 17.4 Captions (Pre-recorded) needs except when media alternative
[Pre-recorded media captions test](https://ictbaseline.access-board.gov/17SyncMedia/#174-test-procedure-for-sc-122-captions-prerecorded) needs "except when the media is a media alternative for text and is clearly labeled as such" to be added.

### PR DESCRIPTION
**Pull request by kengdoj**
closes #382

### CODE REVIEW

File: _baselines/17SyncMedia.md [hunk 1]
```diff
@@ -135,11 +135,32 @@ Evaluation of captions and audio descriptions to assess its equivalence to synch
...
+
 ### Advisory: Tips for streamlined test processes
 
 -   Testing synchronized media is different from testing Baseline [16. Audio-Only and Video-Only content](../16AudioVideo).
 -   Synchronized media players may be software or HTML.
 -   At Level AA, SC 1.2.5 applies to synchronized media. The related Level A requirement, SC 1.2.3, should be marked as ‘Not Tested’ in the test report. It is permissible for test processes to add a test for SC 1.2.3 (evaluate a full text alternative for equivalence). Adding such a test would exceed baseline test requirements and would not affect Baseline 17’s outcome.
+-   All synchronized multimedia should be tested. If the pre-recorded multimedia is labeled as an media alternative for text, confirm that it provides equivalent information as text. If it does not, then it not a media alternative for text. Test the multimedia for captions and audio descriptions
```
Comment (kengdoj) [inline _baselines/17SyncMedia.md:163]:
```suggestion
-   All synchronized multimedia should be tested. If the pre-recorded multimedia is labeled as an media alternative for text, confirm that it provides equivalent information as text. If it does not, then it is not a media alternative for text. Test the multimedia for captions and audio descriptions
```

File: _baselines/17SyncMedia.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ Evaluation of captions and audio descriptions to assess its equivalence to synch
 
 **Baseline Test ID:** 17.4-CaptionsPrerecorded
 #### Identify Content
-<p id="4IC">Pre-recorded synchronized multimedia that is not a media alternative for text and is clearly labeled as such.</p>
+<p id="4IC">Pre-recorded synchronized multimedia that is not clearly labeled as a media alternative for text.</p>
```
Comment (cliffbert) [inline _baselines/17SyncMedia.md:85]: visual and programmatic?

Comment (kengdoj) [reply _baselines/17SyncMedia.md:85]: Yes, your suggestion is much better.

Do you want the visual and programmatic included in the updated statement? The programmatic part would mean "media alternative for text" (or similar) has to be in the accessible name/desc.

Comment (cliffbert) [reply _baselines/17SyncMedia.md:85]: Depends if "labeled" is usually understood to mean those already. Just wanted a way to clarify it if asked. A note would also work.

Comment (kengdoj) [reply _baselines/17SyncMedia.md:85]: I do not think "clearly labeled as such" means the accessible name. I think this is a visual label or description.

This is ACT's *proposed* rule for [Audio element content is media alternative for text](https://www.w3.org/WAI/standards-guidelines/act/rules/afb423/proposed/). It doesn't check programmatic label..

Comment (cliffbert) [reply _baselines/17SyncMedia.md:85]: Then should we specify "Visual" for clarity?

Comment (kengdoj) [reply _baselines/17SyncMedia.md:85]: How about something like this in the Assumptions section for appropriate rules:

Audio-only, video-only, and synchronized media can be "clearly labeled" as alternatives for text in various ways, including text alternatives and text descriptions. At a minimum, a visual label must be provided.

**Comment by kengdoj** [comment]
[Preview of changes](https://federalist-78ccea55-7a06-447c-aabf-bd620b207b74.sites.pages.cloud.gov/preview/atbcb/icttestingbaseline/kengdoj-mediaalt/)

**Pull request merged**